assets/prototype/domain/shared/entities/tickets/predicates/filters/expiredOnly/index.test.ts

```diff
@@ -0,0 +1,30 @@
+import { addWeeks, differenceInMinutes, formatISO, parseISO } from 'date-fns';
+
+import expiredOnly from './index';
+import { nodes as tickets } from '../../../../../../../domain/eventEditor/data/queries/tickets/test/data';
+
+describe('expiredOnly', () => {
+	it('should return an empty array if tickets are archived', () => {
+		const updatedTickets = tickets.map((ticket) => ({ ...ticket, isTrashed: true }));
+		const filteredTickets = expiredOnly(updatedTickets);
+		expect(filteredTickets).toEqual([]);
```

Comment:
I would have expected this to fail since that is not the exact same array and instead would have compared array length against zero... but if this works... then it works! 🤷‍♂

Reply:
The reason I've compared to empty array, but haven't compared to length is to avoid 2 assertions, first to check if it is array and the second to check if it is empty array, because string can also have length zero.

Reply:
That assertion would fail with `toBe`, just double-checked docs: https://jestjs.io/docs/en/expect#toequalvalue